irods/test/scripts/test002.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+. "$BATS_TEST_DIRNAME"/funcs
```

Comment:
I take it `funcs` is a support lib you developed for BATS tests?

Reply:
Yes, commonly needed setup and such.  As to whether to include these tests in 2.1.1 when they're not usable without the content of #508  , I'll defer to others.

Reply:
Consider renaming `funcs` so it's clear to reader what it is for. This can also happen in a separate PR (for 3.0). Just open an issue for it so we don't forget to update it.

As for whether to include the tests - if they aren't directly related to the target issue, pull them out _(this is the general rule)_. If you leave them in, then include words in the commit message body explaining the what/why.

Reply:
`funcs` renamed to `test_support_functions`.